### DISCUSSION TITLE
GUI Design

### DISCUSSION THREAD

**Discussion by mro** [Extended-Thunder/send-later#445] [Ideas]
I saw the screenshot at https://climatejustice.social/@stefanmuelller/108338849200572771 and thought of an earthquake.

You you want to talk about it?

**Comment by jph-sendlater** [comment]
If you have a suggestion I'm happy to hear it.

**Reply by mro** [reply]
drastic reduction and graphical alignment.

I need to know the intention of the screen in one sentence.

Edit: lovely cloister btw.

**Reply by mro** [reply]
are you interested in going further?

**Comment by mro** [comment]
please reply soon or close.
